ci-operator/step-registry/crio-test/agent/test/crio-test-agent-test-ref.yaml

```
@@ -0,0 +1,13 @@
+ref:
+  as: crio-test-agent-test
+  from: assisted-installer-agent-src
```

I guess you meant here:
```suggestion
  from: crio-test-agent-src
```

Otherwise you'll need to import ``assisted-installer-agent-src`` in ``base_images`` stanza